packages/app/src/systems/Send/hooks/useSend.tsx

```diff
@@ -140,17 +142,32 @@ const schemaFactory = (provider?: Provider) =>
               });
             }
 
+            const assets = await AssetService.getAssets();
+            const fuelAssets = await Promise.all(
```

Comment:
It'll already have done fetchAll even before user inputs his password, this is not needed

Reply:
also, "assetsDb" is algo cached, so it won't be missing on any info

Reply:
this will not fetch from server, it will just run the method to get the fuel asset of current chain and do a local verification